sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/AnalyzeSentimentBatchDocumentsAsync.java

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.batch;
+
+import com.azure.ai.textanalytics.TextAnalyticsAsyncClient;
+import com.azure.ai.textanalytics.TextAnalyticsClientBuilder;
+import com.azure.ai.textanalytics.models.AnalyzeSentimentResult;
+import com.azure.ai.textanalytics.models.DocumentResultCollection;
+import com.azure.ai.textanalytics.models.TextAnalyticsRequestOptions;
+import com.azure.ai.textanalytics.models.TextDocumentBatchStatistics;
+import com.azure.ai.textanalytics.models.TextDocumentInput;
+import com.azure.ai.textanalytics.models.TextSentiment;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Sample demonstrate how to analyze sentiment of a batch of text inputs in asynchronously call.
+ */
+public class AnalyzeSentimentBatchDocumentsAsync {
+    /**
+     * Main method to invoke this demo about how to analyze sentiment of a batch of text inputs.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        // Instantiate a client that will be used to call the service.
+        TextAnalyticsAsyncClient client = new TextAnalyticsClientBuilder()
+            .subscriptionKey("<replace-with-your-text-analytics-key-here>")
+            .endpoint("<replace-with-your-text-analytics-endpoint-here>")
+            .buildAsyncClient();
+
+        // The texts that need be analysed.
+        List<TextDocumentInput> inputs = Arrays.asList(
+            new TextDocumentInput("1", "The hotel was dark and unclean. The restaurant had amazing gnocchi.", "en"),
+            new TextDocumentInput("2", "The restaurant had amazing gnocchi. The hotel was dark and unclean.", "en")
+        );
+
+        // Request options: show statistics and model version
+        final TextAnalyticsRequestOptions requestOptions = new TextAnalyticsRequestOptions().setShowStatistics(true);
+
+        // Analyzing batch sentiments
+        client.analyzeBatchSentimentWithResponse(inputs, requestOptions).subscribe(
```

Comment:
Do you use the response headers? I think for samples, we should show use-cases we expect the general populace to use.... analzeBatchSentiment() would be my usual use-case.

Reply:
Currently, we don't have `analyzeBatchSentiment(inputs, requestOptions)` API. So we will add one if needed